.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# Bank Account
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
+
+|  Question 	|   Answer	|
+|---	|---	|
+|   Why is it useful to put classes inside modules?	|   	|
+|   What is accomplished with `raise ArgumentError`	|   	|
+|   Why did we make the `.all` & `.self` methods class methods?  Why not instance methods?	|   	|
+|  Why does it make sense to use inheritance for the different types of accounts?  |  |  
+|   If you had an array of `Account` objects in which some were `SavingsAccount` objects and some were `CheckingAccount` objects could you loop through all the accounts and withdraw money from them?  How would the functionality change?	|   	|
```

Comment:
I think that this question is a bit confusing. Are we trying to get at the fact that each object has a `withdraw` method and therefore it wouldn't matter if they were different types of accounts? Maybe it would help if we actually have them some of the code?